### ISSUE TITLE
Can't recompile on OSX with commit 93a2a480f69e88e5f5301fca3fa34e2f8cac8225

### ISSUE THREAD

**Issue by antoniotuzzi** [VCVRack/Rack#1345]
Operating system: OSX 10 13
Rack version: v1

commit 93a2a480f69e88e5f5301fca3fa34e2f8cac8225
`Remove favorites from settings and favorite scoring in Module Browser.`

after a GIT PULL 

I'm getting these errors:

```src/plugin.cpp:103:50: error: no member named 'st_mtim' in 'stat'
                                plugin->modifiedTimestamp = (double) statbuf.st_mtim.tv_sec + statbuf.st_mtim.tv_nsec * 1e-9;
                                                                     ~~~~~~~ ^
src/plugin.cpp:103:75: error: no member named 'st_mtim' in 'stat'
                                plugin->modifiedTimestamp = (double) statbuf.st_mtim.tv_sec + statbuf.st_mtim.tv_nsec * 1e-9;
                                                                                              ~~~~~~~ ^```

**Comment by antoniotuzzi** [comment]
seems `st_mtim` is declared with another name in the `stat` struct on DARWIN

`st_mtimespec`

if I change the line to
```				plugin->modifiedTimestamp = (double) statbuf.st_mtimespec.tv_sec + statbuf.st_mtimespec.tv_nsec * 1e-9;
```
it works (on OSX)

**Comment by AndrewBelt** [comment]
Fixed on Mac with b5888180beb98ad62b8fb09984c651de1c1d13c1, working on Windows.

**Comment by antoniotuzzi** [comment]
confirmed

**Comment by AndrewBelt** [comment]
Fixed with a6a8826c483f3b68cb03525ca66c15a9b3c2892c on Windows.
